### PR TITLE
Release 20191112

### PR DESCRIPTION
**Pull request by bim-msft**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -628,6 +628,11 @@
       "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/e5ec8997672bc62c8032dd6d4755bf6b8379b3e5/specification/cosmos-db/resource-manager/readme.md",
       "namespace": "Azure::Cosmosdb::Mgmt::V2019_08_01",
       "tag": "package-2019-08"
+    },
+    "azure_mgmt_cosmosdb_2019_08_01_preview_only": {
+      "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/de100e16eabbd5d41551cba8b1d8cd97f3701bf9/specification/cosmos-db/resource-manager/readme.md",
+      "namespace": "Azure::Cosmosdb::Mgmt::V2019_08_01_preview_only",
```
Comment (MyronFanQiu) [inline config/config.json]: We don't need add `_only` into the namespace.

Comment (bim-msft) [reply config/config.json]: Oh. My mistake. Unfortunately, I have just released the gem. Should I release it again after the modification?

Comment (MyronFanQiu) [reply config/config.json]: @bim-msft You need to bump the version number too.

Comment (bim-msft) [reply config/config.json]: @MyronFanQiu Version number has been bumped, please check it again, thanks!

**Pull request merged**